src/bindings/python/wheel/CMakeLists.txt

```diff
@@ -132,23 +132,26 @@ add_custom_command(OUTPUT ${openvino_wheel_path}
     WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
     COMMENT "Building Python wheel ${openvino_wheel_name}"
     VERBATIM)
-
-set(fdupes_report ${CMAKE_CURRENT_BINARY_DIR}/fdupes_report.txt)
-add_custom_command(OUTPUT "${fdupes_report}"
-    COMMAND ${CMAKE_COMMAND}
-        -D Python3_EXECUTABLE=${Python3_EXECUTABLE}
-        -D WORKING_DIRECTORY=${CMAKE_CURRENT_BINARY_DIR}
-        -D WHEEL_VERSION=${WHEEL_VERSION}
-        -D PACKAGE_FILE=${openvino_wheel_path}
-        -D REPORT_FILE=${fdupes_report}
-        -D CMAKE_SHARED_LIBRARY_SUFFIX=${CMAKE_SHARED_LIBRARY_SUFFIX}
-        -P "${CMAKE_CURRENT_SOURCE_DIR}/fdupes_check.cmake"
-    DEPENDS "${openvino_wheel_path}"
-    WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
-    COMMENT "Run 'fdupes' checks for wheel ${openvino_wheel_name}"
-    VERBATIM)
-
-add_custom_target(ie_wheel ALL DEPENDS ${openvino_wheel_path} ${fdupes_report})
+    
+add_custom_target(ie_wheel ALL DEPENDS ${openvino_wheel_path})
+
+if(NOT WIN32)
```

Comment:
why is it needed?

I suppose you can play with warning message:
```cmake
message(WARNING "Failed to find 'fdupes' tool, use 'sudo apt-get install fdupes' to install it")
```

Reply:
Do we expect to use fdupes on Windows? If no, we don't need to this custom target at all